calico/operations/ebpf/install.mdx

```diff
@@ -55,6 +56,18 @@ and in particular, pushing the networking capabilities of the latest Linux kerne
 
 - An underlying network fabric that allows VXLAN traffic between hosts. In eBPF mode, VXLAN is used to forward Kubernetes NodePort traffic.
 
```

Comment:
- IPv6
  :::note
  This feature is tech preview. Tech preview features may be subject to significant changes before they become GA.
  :::

 Limitations:
  - Dual stack is not supported, but ipv4 traffic is allowed into hosts (not workloads) because many managed Kubernetes environments have an ipv4-based control plane.
  - `doNotTrack` policies (xdp) are not supported
  - IPIP is not supported ({{prodname}} iptables does not support it either). VXLAN is the recommended overlay for eBPF mode.

To enable IPv6 in eBPF mode follow [Configure dual stack or IPv6 only](../../networking/ipam/ipv6.mdx). You may be able to run with non-Calico IPAM. eks-cni is known to work.

Reply:
@tomastigera I see OSS docs are not structured like CE docs (where we have 1 requirements section). So you'll need to update the enable doc as well: https://deploy-preview-1113--calico-docs-preview-next.netlify.app/calico/next/operations/ebpf/enabling-ebpf